tests/components/combined_energy/test_config_flow.py

```diff
@@ -0,0 +1,160 @@
+"""Test cases for combined energy config flow."""
+from unittest.mock import patch
+
+import combined_energy.exceptions
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.combined_energy.const import CONF_INSTALLATION_ID, DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+
+
+async def test_form(hass: HomeAssistant):
```

Comment:
I'm not going to continue to comment it on each one, but you need a return type on all of your functions